{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 5",
      "fix_state": "Not affected",
      "package_name": "c-ares",
      "cpe": "cpe:/o:redhat:enterprise_linux:5"
    },
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "c-ares",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "c-ares",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "c-ares",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "nodejs:10/nodejs",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "nodejs:12/nodejs",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "nodejs:14/nodejs",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Not affected",
      "package_name": "rh-nodejs10-nodejs",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Affected",
      "package_name": "rh-nodejs12-nodejs",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Affected",
      "package_name": "rh-nodejs14-nodejs",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2020-11-12T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2020-8277 c-ares: ares_parse_{a,aaaa}_reply() insufficient naddrttls validation DoS",
    "id": "1898554",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1898554"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-119",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2020-8277",
  "document_distribution": "",
  "details": [
    "A Node.js application that allows an attacker to trigger a DNS request for a host of their choice could trigger a Denial of Service in versions \u003c 15.2.1, \u003c 14.15.1, and \u003c 12.19.1 by getting the application to resolve a DNS record with a larger number of responses. This is fixed in 15.2.1, 14.15.1, and 12.19.1."
  ],
  "references": null
}